README.md

```diff
@@ -16,10 +16,42 @@ For this, we use an ADR-inspired process.
 
 To make a meaningful contribution:
 
-1. Create an ADR proposal in the [decisions/log](decisions/log/) with status `In Progress`
-    1. If possible, already create the changes to the technical specification text as well (see above)
-2. Create a Pull Request with the ADR proposal and optionally the specification changes
-3. Once there is consensus on the ADR proposal, the ADR proposal is merged into the `main` branch
+## ADR Submission, Review, and Acceptance Process
+Architecture decisions are submitted via creating a ``Markdown``
```

Comment:
Would advise to be more specific than "Architecture decisions", as there is currently confusion in the community regarding what should be raised as an ADR and what shouldn't be. Would advise to specify as "Architecture decisions and changes to the technical specification".